src/Symfony/Component/Notifier/Tests/Recipient/RecipientTest.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Notifier\Tests\Recipient;
+
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\Notifier\Exception\InvalidArgumentException;
+use Symfony\Component\Notifier\Recipient\Recipient;
+
+/**
+ * @author Jan Schädlich <jan.schaedlich@sensiolabs.de>
+ */
+class RecipientTest extends TestCase
+{
+    public function testCanBeConstructed()
+    {
+        $recipient = new Recipient('test@test.de', '+0815');
+
+        $this->assertSame('test@test.de', $recipient->getEmail());
+        $this->assertSame('+0815', $recipient->getPhone());
+    }
+
+    /**
+     * @dataProvider provideInvalidData
+     */
+    public function testCannotBeConstructedWithInvalidPhone(string $email, string $phone)
```

Comment:
What about also adding a `testCannotBeConstructedWithInvalidEmail()` test?

Reply:
The method name is misleading. I will change it to `testCannotBeConstructedWithInvalidEmailOrPhone()`. The used dataprovider is already covering both cases.